regolith/helpers/u_finishprumhelper.py

```diff
@@ -60,6 +64,16 @@ def db_updater(self):
             print("Please rerun the helper specifying the complete ID.")
             return
         found_projectum.update({'status':'finished'})
+        if rc.pub_sum:
+            found_projectum['deliverable'].update({'public_summary': rc.pub_sum})
```

Comment:
I think the summaries should be at the top level of the schema, not inside the deliverable, which is a milestone.